### ISSUE TITLE
How to find out exactly which labels is used to caculate logits?

### ISSUE THREAD

**Issue by hzpro1221** [mlfoundations/wise-ft#21]
Hi, there are some questions when running your finetune code that i'm facing
In file wiseft/src/model/finetune.py, in line 83, logits is caculated by: `logits = model(inputs)`
`inputs = batch[input_key].cuda()`. I choose fine end to end, so input_key is `image`
 When I print out the shape of logit in logits, it's a 1000 dimenson tensor, so that mean my image is compared with 1000 labels?
Howerver, I got no idea how to find out exactly which label is used to caculate logits
I track back to file wise/src/model/modeling.py, in line 72, so my inputs is run through a image_encoder, the output (logits) is caculated by calling classification_head. Howerver I still don't know which labels are used in this process
`def forward(self, inputs):`
        `if self.process_images:`
            `inputs = self.image_encoder(inputs)`
        `outputs = self.classification_head(inputs)`
        `return outputs`

**Comment by gabrielilharco** [comment]
@hzpro1221 these should be the standard ImageNet labels

**Comment by gabrielilharco** [comment]
You can find the classnames here https://github.com/mlfoundations/wise-ft/blob/master/src/datasets/imagenet_classnames.py
